command/harness/setup.go

```diff
@@ -96,17 +93,17 @@ func HandleSetup(ctx context.Context, r *SetupVMRequest, s store.StageOwnerStore
 	if !poolManager.Exists(pool) {
 		return nil, fmt.Errorf("pool not defined")
 	}
-
-	instance, err := poolManager.Provision(ctx, pool, env.Runner.Name, env.Settings.LiteEnginePath)
+	_, err := s.Find(ctx, stageRuntimeID, pool)
 	if err != nil {
-		return nil, fmt.Errorf("failed provisioning")
+		if err = s.Create(ctx, &types.StageOwner{StageID: stageRuntimeID, PoolName: pool}); err != nil {
```

Comment:
We have err on L97 as well as L98 (local variable). Not sure if that'll create a bug - perhaps we should change the error variable in the if condition?

Reply:
I actually think this is fine?